simplex.m

```diff
@@ -0,0 +1,62 @@
+function[x, g, A] = simplex(A, b, c);
+# Simplex algorithm in standard form:
```

Comment:
The indentation could be changed to improve readability.

*MatlabIndentationBear, severity INFO, section `matlab`.*

The issue can be fixed by applying the following patch:

```diff
--- a/simplex.m
+++ b/simplex.m
@@ -1,14 +1,14 @@
 function[x, g, A] = simplex(A, b, c);
-# Simplex algorithm in standard form:
-# c*x = max
-# A*x <= b
-# x_i > 01
-#
-# Output parameters:
-# A - final tableau of the simplex method
-# subs - indices of the basic variables in the final tableau
-# x - optimal solution
-# z - value of the objective function at x.
+  # Simplex algorithm in standard form:
+  # c*x = max
+  # A*x <= b
+  # x_i > 01
+  #
+  # Output parameters:
+  # A - final tableau of the simplex method
+  # subs - indices of the basic variables in the final tableau
+  # x - optimal solution
+  # z - value of the objective function at x.
   c = -c;
   [m, n] = size(A);
    
```